City-of-Helsinki/outdoors-sports-map
# Add missing outdoor markers

## Description
Add missing markers:
  - adds missing good-on outdoor markers

## Context

[HUL-6](https://andersinno.atlassian.net/browse/HUL-6)

## How Has This Been Tested?

click to see the details of an outdoor service, it should not throw any error and expect to see the details with related good-on icon on the map.

## Manual Testing Instructions for Reviewers

click to see the details of an outdoor service, it should not throw any error and expect to see the details with related good-on icon on the map.

## Screenshots
![cooking-place-good-on](https://user-images.githubusercontent.com/117433931/231174335-769975e6-3f2f-4c20-ad2f-9334255cd704.png)
